metafix/src/test/resources/org/metafacture/metafix/integration/lookup/fromJson/toJson/lookupSimpleWithMultiColumnFile/test.fix

```diff
@@ -0,0 +1,6 @@
+put_filemap("idLookup", "./mapfile.tsv", sep_char:"\t",key_column:"1",value_column:"2",expected_columns:"3")
+put_filemap("nameLookup", "./mapfile.tsv", sep_char:"\t",expected_columns:"3")
```

Comment:
You've got the arguments reversed: `put_filemap("<sourceFile>", "<mapName>", ...)`

Reply:
Thanks.